bokehjs/package.json

```diff
@@ -21,6 +21,7 @@
     "browserify": "^6.2.0",
     "browserify-eco": "^0.2.4",
     "browserify-shim": "^3.8.0",
+    "Canteen": "platfora/Canteen",
```

Comment:
so we are using just master? without any hash??? we need to point to an specific commit...